prototype/app/views/how-it-works.html

```diff
@@ -0,0 +1,24 @@
+{% extends "layouts/main.html" %}
+
+{% set pageTitle = "How it works" %}
+
+{% block content %}
+
+<div class="govuk-width-container ">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">How it works</h1>
```

Comment:
I'm not convinced on this summary - but that's one I can take to the original doc!